cpp/log/logged_entry.h

```diff
@@ -1,7 +1,7 @@
 /* -*- mode: c++; indent-tabs-mode: nil -*- */
 
-#ifndef LOGGED_CERTIFICATE_H
-#define LOGGED_CERTIFICATE_H
+#ifndef LOGGED_ENTRY_H
+#define LOGGED_ENTRY_H
```

Comment:
Hmm, there's no `// LOGGED_ENTRY_H` at the `#endif`? I'm missing some tool that would have told us this, I think. :smiley: